chutney/jira/src/main/java/com/chutneytesting/jira/domain/JiraXrayService.java

```diff
@@ -150,4 +152,18 @@ private String formatEvidenceFilename(String parentStep, String stepName) {
             + (parentStep.trim().isEmpty() ? "" : "_")
             + stepName.trim().replace(" ", "-");
     }
+
+    private boolean loadJiraServerConfiguration() {
+        JiraServerConfiguration currentConfiguration = jiraRepository.loadServerConfiguration();
+        boolean hasConfigurationChanged = !currentConfiguration.equals(this.jiraServerConfiguration);
+        if (hasConfigurationChanged) {
+            LOGGER.info("Jira server configuration has changed");
+            this.jiraServerConfiguration = currentConfiguration;
+        }
+        boolean isConfigurationValid = this.jiraServerConfiguration.isValid();
+        if (hasConfigurationChanged && !isConfigurationValid) {
```

Comment:
Can the configuration be invalid without bieng changed ? If so maybe we can throw the exception